auth/server/src/main/java/com/seagates3/policy/PolicyValidator.java

```diff
@@ -280,6 +281,19 @@ ServerResponse validateStatementSyntax(JSONObject jsonObject) {
               "Statement array element must be a instance of JSONObject");
           return response;
         }
+        JSONObject obj = (JSONObject)arr.get(count);
+        if (obj.has(JsonDocumentFields.STATEMENT_ID)) {
+          String sid = obj.get(JsonDocumentFields.STATEMENT_ID).toString();
+          if (sids.contains(sid)) {
+            response = responseGenerator.malformedPolicy(
+                "Statement IDs (SID) in a single policy must be unique.");
+            LOGGER.error(
```

Comment:
print sid value, along with error message

Reply:
This can be done but  the error msg is in sync with AWS.

Reply:
Fixed, added sid value in LOGGER.error()